README.md

```diff
@@ -30,6 +30,10 @@ We want to get this out and working so we can move on to more significant functi
 
 We are assessing the feature roadmap (contact tracing and more) for the next and subsequent releases.
 
+## Getting Started
+
+See [ONBOARDING](https://github.com/WorldHealthOrganization/app/blob/master/ONBOARDING.md)
```

Comment:
I'd suggest renaming this to CONTRIBUTING.md -- it's a more standard convention for open source repos. See https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file

Reply:
oops, never mind, we already have a CONTRIBUTING.md -- please disregard my comment!

Reply:
definitely a fair comment! I think the idea behind this is that so many people are coming in from all over on this project, so this is meant to be a bit more organizational information beyond just the contributing guidelines. maybe we eventually merge the two 👍